File: src/Nethereum.Generators.JavaScript/codeGenNodeTestsVb/MyContractName/CQS/AllowanceFunction.vb
Imports System
Imports System.Threading.Tasks
Imports System.Numerics
Imports Nethereum.Hex.HexTypes
Imports Nethereum.Contracts.CQS
Imports Nethereum.ABI.FunctionEncoding.Attributes
Imports StandardToken.MyContractName.DTOs
Namespace StandardToken.MyContractName.CQS

    <[Function]("allowance", "uint256")>
    Public Class AllowanceFunction
        Inherits ContractMessage
    
        <[Parameter]("address", "_owner", 1)>
        Public Property Owner As String
        <[Parameter]("address", "_spender", 2)>
        Public Property Spender As String
    
    End Class


End Namespace

File: src/Nethereum.Generators.JavaScript/codeGenNodeTests/MyContractName/CQS/AllowanceFunction.cs
using System;
using System.Threading.Tasks;
using System.Numerics;
using Nethereum.Hex.HexTypes;
using Nethereum.Contracts.CQS;
using Nethereum.ABI.FunctionEncoding.Attributes;
using StandardToken.MyContractName.DTOs;
namespace StandardToken.MyContractName.CQS
{
    [Function("allowance", "uint256")]
    public class AllowanceFunction:ContractMessage
    {
        [Parameter("address", "_owner", 1)]
        public string Owner {get; set;}
        [Parameter("address", "_spender", 2)]
        public string Spender {get; set;}
    }
}
